lib/check_data_points.rb

```diff
@@ -0,0 +1,9 @@
+class CheckDataPoints
+  def self.tipline_requests(team_id, date = nil)
```

Comment:
Replace `date` by two arguments: `from_date` and `to_date`. Both are mandatory.

Reply:
Request: Add a `granularity` argument. Check designs/specs for possible values (daily, weekly, etc.).